package-lock.json

```diff
@@ -1,13 +1,13 @@
 {
```

Comment:
this file should probably not be in git

Reply:
How unreliable it sometimes it, npm says it's supposed to be in git. https://docs.npmjs.com/files/package-lock.json#description

Reply:
in this repo it's probably ok,
but in our other repos that depend heavily on one another it just put huge chunks in diffs each time we update a dep.